sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ConsistencyPolicy.java

```diff
@@ -52,16 +52,21 @@ public ConsistencyPolicy(String jsonString) {
      */
     public ConsistencyLevel getDefaultConsistencyLevel() {
 
-        ConsistencyLevel result = ConsistencyPolicy.DEFAULT_DEFAULT_CONSISTENCY_LEVEL;
-        String consistencyLevelString = super.getString(Constants.Properties.DEFAULT_CONSISTENCY_LEVEL);
-        try {
-            result = ConsistencyLevel
-                         .valueOf(CaseFormat.UPPER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, consistencyLevelString));
-        } catch (IllegalArgumentException e) {
-            // ignore the exception and return the default
-            this.getLogger().warn("Unknown consistency level {}, value ignored.", consistencyLevelString);
+        if (this.cachedConsistencyLevel == null) {
+            ConsistencyLevel result = ConsistencyPolicy.DEFAULT_DEFAULT_CONSISTENCY_LEVEL;
+            String consistencyLevelString = super.getString(Constants.Properties.DEFAULT_CONSISTENCY_LEVEL);
+            try {
+                result = ConsistencyLevel
+                    .valueOf(CaseFormat.UPPER_CAMEL.to(CaseFormat.UPPER_UNDERSCORE, consistencyLevelString));
```

Comment:
please use the following instead:

`ConsistencyLevel.fromServiceSerializedFormat(.)`
if the method is not accessible from this package use:

`BridgeInternal.fromServiceSerializedFormat()`